fly/fly.py

```diff
@@ -8,17 +9,14 @@
 class Fly:
     def __init__(
             self, concourse_url, executable='/usr/local/bin/fly',
-            platform='darwin', target='default'
+            target='default'
     ):
         self.concourse_url = concourse_url
         self.executable = executable
-        self.platform = platform
+        self.platform = platform.system().lower()
```

Comment:
You don't really need to store `self.platform` if you're not allowing the user to specify it - just get `platform.system().lower()` in `get_fly()` when building the URL.